op-node/flags/flags.go

```diff
@@ -139,6 +139,13 @@ var (
 		Required: false,
 		Value:    4,
 	}
+	SequencerFencingCheckEndpoint = &cli.StringFlag{
```

Comment:
Intentional?

Reply:
nope you can ignore the other stuff, just the deployments/deploy-config files